src/GitReportCL/DateAndPathValidation.cs

```diff
@@ -6,27 +6,24 @@ class GitDiffArgumentsValidation
     {
         public bool AreDatesAndPathValid(string[] arguments, GitDiffArguments gitArgument)         
```

Comment:
Usually we don't want to methods modyfiing parameters given to them, it makes code too complex.